static/css/restyle/restyle.less

```diff
@@ -1994,6 +1999,10 @@ body.editor-tools {
         border-radius: 0;
       }
 
+      .results .files {
+        width: 18em;
```

Comment:
> I see you are using the 18em in multiple commits. Maybe it would make sense to define a less variable for the width of the files tree?

That was purely coincidental.